### ISSUE TITLE
Token is not showed after authentication

### ISSUE THREAD

**Issue by ravengalqueen** [dev-sda1/spotify-web-presence#1]
Im using this for my web portfolio, however when it comes the time to authenticate and get the refresh token the console is empty after authenticating, im really hoping you can help me with this, thanks in advance

**Comment by dev-sda1** [comment]
Hey @ravengalqueen ,

Admittedly it's been a while since I've done anything with this repo, but are you able to provide some reproduction steps? I've done a quick refresher on the [instructions provided in the readme](https://github.com/dev-sda1/spotify-web-presence/blob/main/README.md) and had a refresh token output in my console view.

**Comment by ravengalqueen** [comment]
yeah, i made a new directory on my main directory, did the gitclone inside, ran npm install, edited the config.json, and edited the app.js to have the right return URI since i use the port 63342 instead of 1337

**Comment by dev-sda1** [comment]
Does the console return anything specifically, or do you see any sort of error in the json response?

**Comment by ravengalqueen** [comment]
console shows nothing except a warning which is probably unrelated ("Cookie “Webstorm-ca5972d5” with the “SameSite” attribute value “Lax” or “Strict” was omitted because of a cross-site redirect.")

**Comment by ravengalqueen** [comment]
wait for some reason im just now getting a new error when trying to run node . "Error: listen EADDRINUSE: address already in use :::63342", it's weird it never showed that before

**Comment by dev-sda1** [comment]
Are you sure you set up the integration app on the Spotify Developer Portal correctly?

I've added an example below:
<img width="1482" alt="image" src="https://github.com/user-attachments/assets/d7f5ee0a-284c-4666-a188-08dbc83cae35">


> wait for some reason im just now getting a new error when trying to run node . "Error: listen EADDRINUSE: address already in use :::63342", it's weird it never showed that before

Might just be from an existing running session you've forgotten to close. Seems like you're using WebStorm so try closing and reopening the IDE

**Comment by ravengalqueen** [comment]
I tried the repo again and somehow this time it worked, i probably changed something i wasnt supposed to and didnt notice or my file structure was wrong in some way, thank you so much for your help btw, not many people would answer to an issue in a repo they havent touched in 3 years
